README.md

```diff
@@ -471,59 +471,59 @@ Usage: shellspec [options] [files or directories]
 
 ### Configure default options
 
-To change default options for `shellspec` command, create options file.
-Read files in the order the bellows and overrides options.
+To change the default options for the `shellspec` command, create an options
+file. Files are read in the order show below, options defined last take precedence.
```

Comment:
@ko1nksm please confirm that option value read last take precedence.

Reply:
Yes, options defined last take precedence.